### ISSUE TITLE
Spam seems to be posted before it is deleted

### ISSUE THREAD

**Issue by xibe** [strider72/spam-karma#33]
Using 2.4-alpha-20150612.

For the past month, I've receive comment notification from my main blog (the usual, "New comment on your post #324 "Livre d'or"") which was clearly spam. 

Clicking the post's link reveals no commit: it has already been deleted.

Could SK be publishing comments before deleting this?

I have not changed anything in my settings recently.

**Comment by strider72** [comment]
That's how SK has worked for me for a long time.  Predates Dr Dave open sourcing it, I believe.

**Comment by xibe** [comment]
Sorry for the ultra late answer.

As you know, I've been using SK and SK2 for a heck of a long time, and only recently has it shown this behavior.
Can't see any option to change this. Could this idea be considered?

**Comment by strider72** [comment]
It's a bug, not an intentional feature. (Note I marked this Issue "bug".) I agree it should be fixed.  But SK 2.3 has been doing this to me for a long time. Must be intermittent, or only happen on certain setups.

**Comment by strider72** [comment]
xibe —  Did this issue specifically start when up upgraded to 2.4a ?

**Comment by xibe** [comment]
Still using v2.4-alpha-20150612.
I do think it started happening once I switched from my legacy SK2 plugin to the GitHub- (or was it GCode-?) hosted one.

**Comment by strider72** [comment]
Yeah, I asked because — come to think of it — I've not been on regular 2.3 for a long time; I've been on variations of 2.4a. I can't honestly recall when it started for me.

**Comment by xibe** [comment]
I guess I should switch back to 2.3 and see how that works for me.
